templates/zookeeper.service.j2

```diff
@@ -11,6 +11,7 @@ ExecStart={{ zookeeper_dir }}/bin/zkServer.sh start
 ExecStop={{ zookeeper_dir }}/bin/zkServer.sh stop
 SuccessExitStatus=143
 Restart=on-failure
+RestartSec=5
```

Comment:
Set a 5 second delay between restarts as if too many restarts occur within a specified time period then systemd will not attempt to start the process again and it needs to be reset.